tests/test_cloudwatch.py

```diff
@@ -49,3 +50,53 @@ def test_read_logs(loggroup):
     )
     assert len(df.index) == 5
     assert len(df.columns) == 3
+
+
+def test_describe_log_streams_and_filter_log_events(loggroup):
```

Comment:
It would be good to also have tests for the cases where we want the API call to throw an error, like `InvalidArgumentCombination`.

Reply:
I've added two additional `InvalidArgumentCombination` cases for `describe_log_streams` and `filter_log_events`